content/components/action-menu.mdx

```diff
@@ -8,40 +8,65 @@ import {Box, Button, Heading, Link} from '@primer/react'
 import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
 
-## Usage
+<Note>Multi level is not available yet in the React implementation.</Note>
 
-Action menu is used for actions, navigation, to display secondary options, or single/multi select lists. They appear when users interact with buttons, actions, or other controls.
+## Usage
 
-## Anatomy
+An action menu is a list of items, with each item representing an action, command, or current selection, which can be a single or multi-select.
 
 <img
   width="960"
-  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
-  src="https://user-images.githubusercontent.com/18661030/226487158-06c6cc38-65de-438c-8028-373862fa0dfe.png"
+  alt=""
+  src="https://user-images.githubusercontent.com/980622/233374605-cee72894-fc5e-46f4-a1ea-e82aa26084ef.png"
 />
 
-## Options
+There are different ways to use menus, such as:
 
-<img
-  src="https://user-images.githubusercontent.com/18661030/226486461-c43f75c5-d9ed-415f-907c-9d2dbf71e3a2.png"
-  width="960"
-  alt=""
-/>
-<Caption>
-  The trigger button can be a standard or icon button. For single select menus that display the selection in the
-  trigger, a label must be persist either internally or externally.
-</Caption>
+- **Button**: Shows items directly related to the current view.
+- **Button (Selection)**: Shows items directly related to the button, with a title prefix, like "Sort by: Title."
+- **Context menu**: Often contains multiple levels and tends to be only available through right clicking items or through [icon buttons](/components/icon-button) that appear on hover.
```

Comment:
I'm having a hard time comprehending this bit. The first two items are referring to how the trigger button works, and the last feels like a different thing. I wonder if it could be rephrased to talk about different _kinds_ of menus? Simple action, selection, or multi level?

Reply:
Agreed. I've rewritten this whole section and created a new section in `options` to stay focused on the action list primitives that can be used within this menu.

Let me know what you think.